recipes/cuquantum/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "0.1.0" %}
+{% set patch_version = "30" %}
+
+package:
+  name: cuquantum
+  version: {{ version }}.{{ patch_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-x86_64-{{ version }}.{{ patch_version }}-archive.tar.xz   # [linux64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-sbsa-{{ version }}.{{ patch_version }}-archive.tar.xz     # [aarch64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-ppc64le-{{ version }}.{{ patch_version }}-archive.tar.xz  # [ppc64le]
+
+  sha256: 8ad8e98f14275ffe0de02574be5c86224af1c657c41baf02c16440301ffe0aae  # [linux64]
+  sha256: 3dd04cf08f1323318e0e63a7e28bc904c426ced3367aca809fba7e7beef94063  # [aarch64]
+  sha256: 0eb84eef619a1cfab5870fb585200a9869a197866685252e4ca8187322809554  # [ppc64le]
+
+build:
+  number: 1
+  skip: True  # [win or ppc64le or cuda_compiler_version != "11.2"]
```

Comment:
Please send a PR to add a cuda 11.2 job to staged-recipes so that we can properly verify this PR

Reply:
Hi @isuruf sure, my understanding from the past experience is the verification is postponed until the package is actually being generated in the feedstock (which we've done for this case). Please share instructions with me for how to add a new CUDA job and I'll give it a shot, thanks!

Reply:
I don't think that's the case. See your comment at https://github.com/conda-forge/staged-recipes/pull/14276#issuecomment-802046404

Reply:
I am confused. In my earlier comment I did say
> which is fine because by the time the feedstock is created, the new settings from conda-forge-pinning will kick in IIUC. Nothing to worry.

so what am I missing here? 

Anyway, could you share pointers for how to proceed here @isuruf?

Reply:
I sent a PR. https://github.com/conda-forge/staged-recipes/pull/17570. So all you have to do is rebase.

Reply:
Got it, thanks for moving ahead of me Isuru, will do.

Reply:
I see that CUDA 11.x CIs are kicked off now. Mark as resolved.